_posts/18-05-2022-a_story_unfolds.md

```diff
@@ -0,0 +1,202 @@
+---
+layout: post
+title: A story unfolds
+category: Haskell
+description: or how I tried to plant a tree
+---
+# A story unfolds
+
+### or how I tried to plant a tree
+
+I tried to re-implement the Unix `tree` tool in Haskell, as a learning exercise. The problem is well-defined, well-structured, obviously recursive, so it should be easy to do, a no-brainer almost[^excuses].
```

Comment:
> The problem is well-defined

`tree` tool isn't as familiar to most people as you might expect... just a screenshot/snippet with the output to define what `tree` does would be useful!

Reply:
Done.